src/editor/block/commands.js

```diff
@@ -47,6 +47,25 @@ export const addNewBlockAfterCurrent = ({ state, dispatch }) => {
     return true;
 }
 
+export const addNewBlockUpTop = ({ state, dispatch }) => {
+    if (state.readOnly)
+        return false
+    const block = state.facet(blockState)[0]
```

Comment:
It seems there is always at least one block in the buffer so this should work. I could not make it break by fiddling around deleting everything in the editor etc.